docs/API.md

```diff
@@ -31,16 +41,21 @@ reporter.add_examiner examiner
 reporter.show
 ```
 
-This will show the list of errors in variable `source`.
-
-`Reek::Examiner.new` can take `source` as `String`, `File` or `IO`.
+This would output on STDOUT:
 
 ```
-# Examine a file object
-reporter.add_examiner Reek::Examiner.new(File.new('dirty.rb'))
+dirty.rb -- 4 warnings:
+  [2]:Dirty#m has the name 'm' (UncommunicativeMethodName)
+  [2]:Dirty#m has the parameter name 'a' (UncommunicativeParameterName)
+  [2]:Dirty#m has the parameter name 'b' (UncommunicativeParameterName)
+  [2]:Dirty#m has unused parameter 'c' (UnusedParameters)
 ```
 
-Also, besides normal text output, `reek` can generate output in YAML,
+Note that `Reek::Examiner.new` can take `source` as `String`, `Pathname`, `File` or `IO`.
```

Comment:
I think we agreed in https://github.com/troessner/reek/pull/596#discussion_r34437240 and follow-up discussion to remove the Pathname option again?